### ISSUE TITLE
Misspelled assassin

### ISSUE THREAD

**Issue by karlek** [Lapsang-boys/Rex-Temporis#1]
```shell
# wurst/objects/abilities/towers/BrotherhoodOfBandits/Assassination.wurst
94:function isAssasain() returns bool
113:    breakStealthTrigger.addCondition(Condition(function isAssasain))

# wurst/objects/abilities/towers/BrotherhoodOfBandits/AssassinationOld.wurst
29://         this.trig.addCondition(Condition(function isAssasain))
48:// // Check if assasain
49:// function isAssasain() returns bool
```

😂

**Comment by godbit** [comment]
Rofl.. Fix i eftermiddag

**Comment by godbit** [comment]
Fixed through ^ and commit 0867edb05fa50227fa552de143f46041fb0e514e

Closing!
